locale/ru/messages.json

```
@@ -1,239 +1,239 @@
+  "Stick sensitivity": "Чувствительность управления стиком",
+  "Select stick sensitivity": "Выберите чувствительность управления стиком",
+  "Motion sensitivity": "Чувствительность управления движением",
+  "Select motion sensitivity": "Выберите чувствительность управления движением",
+  "About": "О сайте",
+  "Feedback": "Обратная связь",
+  "Noticed a bug? Something could be done better? Thought of a new cool feature you'd like to see on the site? Need help using the site? You can either:": "Заметили ошибку? Что можно было бы сделать лучше? Подумали о новой классной функции, которую вы хотели бы увидеть на сайте? Нужна помощь в использовании сайта? Вы можете либо",
   "Create an issue on <0>GitHub</0>": "",
-  "Post on the #helpdesk or #feedback channel of our <0>Discord</0>": "",
+  "Post on the #helpdesk or #feedback channel of our <0>Discord</0>": "Сообщение на #helpdesk или #канал обратной связи из наших <0>Дискорд</0>",
   "DM me on <0>Twitter</0> or Discord (Sendou#0043)": "",
-  "Thanks to": "",
-  "provided the data for Top 500 X Rank and League": "",
-  "provided background pictures of 4v4 maps for the map planner": "",
-  "provided the data for tournament browser": "",
-  "gave plenty of useful feedback": "",
-  "art for the site - front page inkling, icons and footer drawing": "",
+  "Thanks to": "Благодаря",
+  "provided the data for Top 500 X Rank and League": "предоставил данные для Топ-500 X ранга и лиги",
+  "provided background pictures of 4v4 maps for the map planner": "предоставил фоновые изображения карт 4v4 для планировщика карт",
+  "provided the data for tournament browser": "предоставил данные для браузера турнира",
+  "gave plenty of useful feedback": "дал много полезных отзывов",
+  "art for the site - front page inkling, icons and footer drawing": "искусство для сайта - надпись на первой странице, значки и рисунок нижнего колонтитула",
   "<0>Contributions to the codebase</0>": "",
-  "translation": "",
-  "Patch {CURRENT_PATCH}": "",
-  "AP = Ability Point = Mains * 10 + Subs * 3": "",
-  "Hide settings": "",
-  "Show settings": "",
-  "Hide stats at base value": "",
-  "Progress bars show progress to max value": "",
-  "Start charts Y axis from zero": "",
-  "Hide Stats": "",
-  "Show Stats": "",
-  "Add build": "",
-  "builds": "",
-  "hidden": "",
-  "Edit free agent post": "",
-  "New free agent post": "",
-  "Your free agent post is about to expire": "",
-  "Free agent posts that haven't been updated in over a month will be hidden. Please press the button below if you are still a free agent.": "",
-  "I'm still a free agent": "",
-  "All ({0})": "",
-  "Frontline ({0})": "",
-  "Support ({0})": "",
-  "Backline ({0})": "",
-  "Once you match with other free agents they are shown here.": "",
+  "translation": "перевод",
+  "Patch {CURRENT_PATCH}": "Патч {ТЕКУЩИЙ_ПАТЧ} ",
```

```suggestion
  "Patch {CURRENT_PATCH}": "Патч {CURRENT_PATCH} ",
```

Here the value inside curly brackets should be left untranslated.